.github/workflows/build.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
It seems no check is running for this PR.
I guess you should fix the branch names in `on` conditions?

Reply:
Or removing branches from `pull_request` might be good.

Reply:
Now the CI works on this PR! 0246885